src/test-e2e/kotlin/com/buildit/bookit/v1/booking/BookingTests.kt

```diff
@@ -48,7 +48,9 @@ class `Booking E2E Tests` {
         inner class `valid` {
             private val expectedBooking = """
                         {
-                            "bookableId": "aab6d676-d3cb-4b9b-b285-6e63058aeda8",
+                            "bookable": {
+                                "id": "aab6d676-d3cb-4b9b-b285-6e63058aeda8"
+                            },
```

Comment:
@frostiebot @lizziesz  one example of how the shape of responses change...generally if you had an id coming back now you have a whole object (we only validate the "id" but everything comes back...)...

we can have further discussions as to whether this is OK or whether we shouldn't do that... or whether we should replace ids w/ URIs etc etc

Reply:
This allows for flexibility in future and is better now before things harden more (IMO).